ci/tests/vmachine/vmachine_test.py

```diff
@@ -104,16 +121,38 @@ def create_machine_from_existing_raw_disk(machine_name, vpool_name, disk_name):
         logger.error("Error while creating vmachine: {0}".format(err))
 
 
-def remove_machine_by_name(vmachine_name):
-    logger.info("Removing {0} vmachine".format(vmachine_name))
-    out, err = general.execute_command('virsh destroy {0}'.format(vmachine_name))
+def shut_down_vmachine_by_name(machine_name):
+    logger.info("Shutting down vmachine {0}".format(machine_name))
+    out, err = general.execute_command('virsh destroy {0}'.format(machine_name))
+    if err:
+        logger.error("Error while shutting down vmachine {0}: {1}".format(machine_name, err))
+
+
+def start_vmachine_by_name(machine_name):
+    logger.info("Starting vmachine {0}".format(machine_name))
+    out, err = general.execute_command('virsh start {0}'.format(machine_name))
     if err:
-        logger.error("Error while stopping vmachine: {0}".format(err))
+        logger.error("Error while shutting down vmachine {0}: {1}".format(machine_name, err))
+
+
+def undefine_machine(vmachine_name):
     out, err = general.execute_command('virsh undefine {0}'.format(vmachine_name))
     if err:
         logger.error("Error while removing vmachine: {0}".format(err))
 
 
+def remove_machine_by_name(vmachine_name):
```

Comment:
Same comments apply here for undefine and remove_machine
Also try to use consistent names
Here its a 'machine', in the other functions its a 'vmachine'
it should all be 'vmachine'